data/brands/amenity/restaurant.json

```diff
@@ -3181,6 +3181,19 @@
         "name": "Pizza Paï"
       }
     },
+    {
+      "displayName": "Pizza Pilgrims",
+      "locationSet": {"include": ["gb"]},
+      "tags": {
+        "amenity": "restaurant",
+        "brand": "Pizza Pilgrims",
+        "brand:wikidata": "Q110454959",
+        "cuisine": "pizza",
+        "diet:vegan": "yes",
```

Comment:
I'd remove the "_diet_" tag as I think we're still focusing on just name / branding at the moment. #4855